ngraph/frontend/onnx_import/src/op/div.hpp

```diff
@@ -68,7 +71,7 @@ namespace ngraph
                                                                     node.get_ng_inputs().at(1))};
                 }
 
-            } // namespace set_1
+            } // namespace set_7
```

Comment:
Something's wrong here - this line closes the `set_1` namespace, not 7.

Reply:
When clang-format applied this I was surprised as well, so I double checked that .. look [here](https://github.com/openvinotoolkit/openvino/blob/7cf7e1b5fa6a6a4371b9ef60776b2c67ba95490c/ngraph/frontend/onnx_import/src/op/div.hpp#L61-L71).